bin/gollum-migrate-tags

```diff
@@ -20,11 +20,11 @@ end
 
 opts = OptionParser.new do |opts|
 opts.banner = <<EOF
-Use this tool to migrate a wiki repository created under Gollum versions 4.x or earlier to a 5.x compatibile repo.
+Use this tool to migrate a wiki repository created under Gollum versions 4.x or earlier to a 5.x compatible repo.
 It finds and repairs Gollum link tags that no longer work under 5.x for three reasons:
 
 * 5.x wiki internal links may contain spaces. [[Bilbo Baggins]] and [[Bilbo-Baggins]] therefore link to distinct pages.
-* 5.x wiki internal links are case senitive
+* 5.x wiki internal links are case sensitive
```

Comment:
Fwiw, you use `case-insensitively` below, in general it's probably best to use the same style for both `case-(in)sensitive`. But I'm not actively enforcing style (and am still thinking about whether I want to flag the space instead of a hyphen as a general thing for this word pair, although I am leaning towards doing so...